netstatus/dashboard/templates/dashboard/charts.html

```diff
@@ -0,0 +1,305 @@
+{% include 'dashboard/_header.html' %}
+{% load staticfiles %}
+
+<script>
+    var total_risk_levels = 4;
+    var risk_level_labels = ['Neutral', 'Low', 'Moderate', 'High'];
+    var colors = {
+        'Low': '#006e9c',
+        'Neutral': '#3f9c35',
+        'Moderate': '#ec7a08',
+        'High': '#c00',
+        'Total': '#999'
+    };
+</script>
+
+<div class="container-fluid container-pf-alt-nav-pf-vertical-alt">
+    <div class="page-header">
+        <h1>Charts</h1>
+    </div>
+
+    <div class="row">
+        <div class="col-lg-6 col-sm-12">
+            <h3>Last 10 minutes</h3>
+        </div>
+
+        <div class="col-lg-6 col-sm-12">
+            <h3>New flows by hour of day</h3>
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="col-lg-6 col-sm-12">
+            <div id="tenMinChart" class="line-chart-pf"></div>
+            <script>
+                var tenMinChartLabels = ['x'].concat(risk_level_labels).concat(['Total']);
```

Comment:
tenMinChartLabels defines labels of series we'll provided. `x` means that we'll be giving value of x (instead of default sequence of integers):

`labels = ['x', 'neutral', 'low', 'high']`
points:
`['2016-06-20T10:00:00Z', 0, 1, 2]
['2016-06-20T10:01:00Z', 1, 2, 2]
['2016-06-20T10:02:00Z', 1, 4, 3]`